torch/_torch_docs.py

```diff
@@ -7999,18 +7999,47 @@ def merge_dicts(*dicts):
            r"""
 svd(input, some=True, compute_uv=True, *, out=None) -> (Tensor, Tensor, Tensor)
 
-This function returns a namedtuple ``(U, S, V)`` which is the singular value
-decomposition of a input real matrix or batches of real matrices :attr:`input` such that
-:math:`input = U \times diag(S) \times V^T`.
+Compute the singular value decomposition of either a matrix or batch of
+matrices :attr:`input`." The singular value decomposition is represented as a
+namedtuple ``(U, S, V)``, such that :math:`input = U \times diag(S) \times
+V^T`, where :math:`V^T` is the transpose of ``V``. If the inputs are batches, then
+returns batched outputs for all of ``U``, `S`` and ``V``.
 
-If :attr:`some` is ``True`` (default), the method returns the reduced
-singular value decomposition i.e., if the last two dimensions of
-:attr:`input` are ``m`` and ``n``, then the returned `U` matrix will
-contain only :math:`min(n, m)` orthonormal columns and the size of `V`
-will be :math:`(*, n, n)`.
+The original tensor can be reconstructed by::
```

Comment:
"The original tensor can be reconstructed by" -> "When :attr:\`input\` is a tensor, it can be reconstructed by"